WordPress/wordpress-playground
# Route requests for missing static files using remote asset metadata - Second attempt

## Motivation for the change, related issues

Prior to this PR, we could not easily tell whether we should request a
missing static asset from the web or delegate a request for a missing file
to WordPress.

Related to #1365 where we need better information to judge whether to
handle a request for a missing static file as PHP.

## Implementation details

This PR:
- Includes a list of remote asset paths with minified WP builds
- Updates request handling to only handle missing files as static when
they are explicitly listed as remote assets
(perhaps this should be split into multiple PRs)
- Updates request routing to behave more like a regular web server

By including a list of remote asset paths with minified WP builds, we
can judge which missing static file paths represent remote assets and
which are truly missing and need to be delegated to WordPress.

## Testing Instructions (or ideally a Blueprint)

- CI tests
- Test manually with `npm run dev` and observe that Playground loads
normally with no unexpected errors in the console
